app/graphql/types/invoices/applied_taxes/object.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Types
+  module Invoices
+    module AppliedTaxes
+      class Object < Types::BaseObject
+        graphql_name 'InvoiceAppliedTax'
+
+        field :id, ID, null: false
```

Comment:
A possible approach to share some code between all those `AppliedTaxes` could be to use GraphQL interfaces:

```ruby
module Types
  module Taxes
    module AppliedTax
      include Types::BaseInterface

      field ...
    end
  end
end
```

And then use them in the different types:

```ruby
module Types
  module Invoices
    module AppliedTaxes
      class Object < Types::BaseObject
        graphql_name 'InvoiceAppliedTax'
        implements Types::Taxes::AppliedTaxes

        field :invoice, Types::Invoices::Object, null: false
      end
    end
  end
end
```